Ska/engarchive/derived/orbit.py

```diff
@@ -83,7 +83,7 @@ def arccos_2pi(arg, reflect):
         try:
             # Check if arg is a non-scalar numpy array
             len(arg) and isinstance(arg, np.ndarray)
-        except:
+        except Exception:
```

Comment:
I know this is not pep8/lin related, but this being a generic exception I started thinking what exception it should be. I am guessing it should be TypeError.

I went to check in the file, and it seems to me that the statement `and isinstance(arg, np.ndarray)` does nothing.

Reply:
Good catch, you're right.

Reply:
Fixed.